types/meteor/globals/accounts-base.d.ts

```diff
@@ -146,3 +146,75 @@ declare module Accounts {
 declare module Accounts {
     function onLogout(func: (user: Meteor.User, connection: Meteor.Connection) => void): void;
 }
+
+declare module Accounts {
+    interface LoginMethodOptions {
+        methodName?: string;
+        methodArguments?: any[];
+        validateResult?: Function;
+        userCallback?: (err: any) => void;
+    }
+
+    /**
+     *
+     * Call a login method on the server.
+     *
+     * A login method is a method which on success calls `this.setUserId(id)` and
+     * `Accounts._setLoginToken` on the server and returns an object with fields
+     * 'id' (containing the user id), 'token' (containing a resume token), and
+     * optionally `tokenExpires`.
+     *
+     * This function takes care of:
+     * - Updating the Meteor.loggingIn() reactive data source
+     * - Calling the method in 'wait' mode
+     * - On success, saving the resume token to localStorage
+     * - On success, calling Accounts.connection.setUserId()
+     * - Setting up an onReconnect handler which logs in with
+     *   the resume token
+     *
+     * Options:
+     * - methodName: The method to call (default 'login')
```

Comment:
Options descriptions here can be moved right into the LoginMethodOptions, e.g.

```ts
interface LoginMethodOptions {
    /** The method to call (default 'login') */
    methodName?: string;

    // ...
}
```

Reply:
Done!